src/Components/Admin/navbar.js

```diff
@@ -49,39 +54,56 @@ export default function Navbar({ home }) {
-                            transformOrigin={{
-                                vertical: "top",
-                                horizontal: "right",
-                            }}
-                            open={open}
-                            onClose={handleClose}
-                            className={classes.menu}
-                        >
-                            <MenuItem onClick={handleClose}>
-                                Documentation
-                            </MenuItem>
-                            <MenuItem onClick={handleClose}>Settings</MenuItem>
-                            <MenuItem onClick={handleClose}>Logout</MenuItem>
-                        </Menu>
-                    </div>
+                    {user && (
+                        <div className={classes.profileEnd}>
+                            <IconButton
+                                aria-label="account of current user"
+                                aria-controls="menu-appbar"
+                                aria-haspopup="true"
+                                onClick={handleMenu}
+                                color="inherit"
+                            >
+                                <AccountCircle />
+                            </IconButton>
+                            <Menu
+                                id="menu-appbar"
+                                anchorEl={anchorEl}
+                                anchorOrigin={{
+                                    vertical: "bottom",
+                                    horizontal: "center",
+                                }}
+                                keepMounted
+                                transformOrigin={{
+                                    vertical: "top",
+                                    horizontal: "right",
+                                }}
+                                open={open}
+                                onClose={handleClose}
+                                className={classes.menu}
+                            >
+                                <MenuItem onClick={handleClose}>
+                                    Documentation
+                                </MenuItem>
+                                <MenuItem onClick={handleClose}>
+                                    Settings
+                                </MenuItem>
+                                <MenuItem
+                                    onClick={async () => {
+                                        handleClose();
+                                        history.push("/login");
+                                        auth.signOut();
+                                        await httpGet(
+                                            process.env
+                                                .REACT_APP_ADMIN_BASE_ENDPOINT +
+                                                "admin_logout"
+                                        );
+                                    }}
```

Comment:
I would prefer to have this pulled into another function than make it inline with `onClick`